File: OOP.js
class Game {
    // Creating an instance of a Game
    constructor(gameTitle, gameGenre, gameCharacters = false) { // Using a default value for gameCharacters if none specified
        this.title = gameTitle;
        this.genre = gameGenre;
        this.characters = gameCharacters;
    }

    // Define methods for the Game class here.
    getTitle() {
        return this.title;
    }
}

// Creating instances of Games:
var myGame1 = new Game("Breath of the Wild","RPG",["Link","Zelda","Ganon","Urbosa"]);
var myGame2 = new Game("Tetris","Puzzle",["Mr. Tetronimo", "T block"]);

console.log(myGame1.getTitle());
console.log(myGame1.genre); // Can access a property of this game directly

File: OOP.py
class Game(): # Define a game class
    def __init__(self, game_title, genre_name, list_chars):
        self.title = game_title
        self.genre = genre_name
        self.characters = list_chars

    # Methods for the Game class go here.
    def printTitle(self): 
        print(self.title)
        return self

# Create two instances of a game
my_game = Game("Breath of the Wild", "RPG", ["Link","Zelda","Ganon"])
my_game2 = Game("Randomness","Void",["Chicken", "Mr. Null", "Narrator"])

my_game.printTitle()


